src/Clio.Requests/RequestFinder.cs

```diff
@@ -95,5 +111,13 @@ string TryExpression (Regex expression, CommitInfo commit)
 				return match.Groups[1].Value;
 			return null;
 		}
+
+		string TryBodyExpression (Regex expression, string body)
+		{
+			var match = expression.Match (body);
+			if (match.Success)
+				return match.Groups[1].Value;

```

Comment:
Why `Groups[1]`? What's the `1`?
Also minor whitespace nit: `match.Groups [1].Value`

Reply:
It's the matching group of:

```
Regex BackportExpression = new Regex (@"Backport of #(\d*)", RegexOptions.Compiled);
```

Being the backport number in this case.

0 being the entire string I believe.
Clio's formatting is not standard, I really should add an editor config and force format everything.